packages/flutter/lib/src/widgets/single_child_scroll_view.dart

```diff
@@ -57,12 +58,33 @@ class SingleChildScrollView extends StatelessWidget {
     );
   }
 
+  /// The axis along which the scroll view scrolls.
+  ///
+  /// Defaults to [Axis.vertical].
   final Axis scrollDirection;
 
+  /// Whether the scroll view scrolls in the reading direction in the
```

Comment:
No, I guess not... this is confusingly worded right now, though I think it's technically correct.